math/FunMath.cpp

```diff
@@ -1 +1,13 @@
 #include "FunMath.h"
+
+//Kyle Schultz, Mitch Massey, Tyler Albee, Liz Parker
+
```

Comment:
Add a function comment that says what this function does, what the parameters/return values are

Reply:
You can also remove your names from this file.